server/views/courses/_locationsTable.njk

```diff
@@ -0,0 +1,31 @@
+{% from "govuk/components/table/macro.njk" import govukTable %}
+
+{{ govukTable({
+  attributes: {
+    "data-module": "moj-sortable-table"
+  },
+  head: [
+    {
+      text: "Prison",
+      attributes: {
+        "aria-sort": "ascending"
+      }
+    },
+    {
+      text: "Category",
+      attributes: {
+        "aria-sort": "none"
+      }
+    },
+    {
+      text: "Region",
+      attributes: {
+        "aria-sort": "none"
+      }
+    },
+    {
+      text: ""
```

Comment:
Did we want to give this an `Action` heading or anything similar?

Reply:
Oh yeah, I asked about that in a couple of channels and forgot to update it! What's your preference out of:
- keeping it visually like the design, but with a `govuk-visually-hidden` heading
- adding "Action", against the design?

Also need to add more visually hidden information on each of the links, e.g. the prison name

Reply:
I'd probably opt for the latter, but maybe one to ask Chanelle?